### PR TITLE
bump(x509-certificate-exporter): release 1.10.0, app 2.8.0

### PR DESCRIPTION
**Pull request by npdgm**

### CODE REVIEW

File: charts/x509-certificate-exporter/templates/deployment.yaml [hunk 1]
```diff
@@ -77,6 +77,9 @@ spec:
       {{- if .Values.secretsExporter.debugMode }}
         - --debug
       {{- end }}
+      {{- if $.Values.exposeRelativeMetrics }}
+        - --expose-relative-metrics
+      {{- else }}
```
Comment (jfcoz) [inline charts/x509-certificate-exporter/templates/deployment.yaml:82]: should be {{- end }}

File: charts/x509-certificate-exporter/templates/daemonset.yaml [hunk 2]
```diff
@@ -67,6 +67,9 @@ spec:
       {{- with default $.Values.hostPathsExporter.debugMode $dsDef.debugMode }}
         - --debug
       {{- end }}
+      {{- if $.Values.exposeRelativeMetrics }}
+        - --expose-relative-metrics
+      {{- else }}
```
Comment (jfcoz) [inline charts/x509-certificate-exporter/templates/daemonset.yaml:72]: should be {{- end }}

**Review by jfcoz — COMMENTED** [review]
two `{{ - else }}` should be `{{ - end }}`

File: charts/x509-certificate-exporter/templates/deployment.yaml (hunk 1 above)
Comment (npdgm) [reply charts/x509-certificate-exporter/templates/deployment.yaml:82]: Oh this isn't good at all, thanks for catching it. Looks like it wasn't tested...
We should add a helm linter here :confused:

Comment (npdgm) [reply charts/x509-certificate-exporter/templates/deployment.yaml:82]: Fixed in 1.10.1

**Pull request merged**